package.json

```diff
@@ -138,7 +139,7 @@
     "yuidocjs": "0.10.2"
   },
   "engines": {
-    "node": ">= 0.10.0"
+    "node": "0.12.* || 4.5.* || 6.*"
```

Comment:
Actually I forgot that was a possibility. My goal was to set the lower bound (4.5.0) but I missed that the upper bound could be anything less than `5`.